### PR TITLE
Dev-3102 Additional Details Contracts

### PR DESCRIPTION
**Pull request by jonhill13**
**High level description:**

Updates to the Additional Details section to new style.
Updates to the Additional Details Contract section data.

**Technical details:**

Accordion now handles links, addresses, and lists
Remove Recipient Details
Accordion renders divs

**JIRA Ticket:**
[DEV-3102](https://federal-spending-transparency.atlassian.net/browse/DEV-3102)

**Mockup:**
https://bahdigital.invisionapp.com/share/CZIAB1EVH75#/screens/296001046_Award_Summary_2-0_-_Contract_-_Expanded

The following are ALL required for the PR to be merged:
- [x] Code review
- [ ] All `componentWillReceiveProps`, `componentWillMount`, and `componentWillUpdate` in relevant child/parent components/containers replaced with [future compatible life-cycle methods](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) per [JIRA item 3341](https://federal-spending-transparency.atlassian.net/browse/DEV-3341)
- [x] Design review (@AGuyNamedMarco )
- [ ] Verified cross-browser compatibility

### CODE REVIEW

**Comment by AGuyNamedMarco** [comment]
@jonhill13 Crushed it.  TY sir.  Approved.

File: src/js/dataMapping/awardsv2/additionalDetailsFinancialAssistance.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * additionalDetailsFinancialAssistance.js
+ * Created by Jonathan Hill 09/19/19
+ */
+
+
+const additionalDetailsContracts = () => {};
+
+export default additionalDetailsContracts;
```
Comment (MaxwellKendall) [inline src/js/dataMapping/awardsv2/additionalDetailsFinancialAssistance.js]: I think we can just delete this file?

Comment (jonhill13) [reply src/js/dataMapping/awardsv2/additionalDetailsFinancialAssistance.js]: This was just a place holder for the following PR for financial assistance additional details which i apparently named the file right but not func.

I one file or three does not matter to me. I see benefits both ways.

Comment (MaxwellKendall) [reply src/js/dataMapping/awardsv2/additionalDetailsFinancialAssistance.js]: Cool. I'm good with either way too.

**Pull request merged**